src/pages/distroSettings/tabs/ProviderTab/ProviderTab.tsx

```diff
@@ -1,12 +1,34 @@
 import { useMemo } from "react";
+import { DistroSettingsTabRoutes } from "constants/routes";
+import { useSpruceConfig } from "hooks";
+import { useDistroSettingsContext } from "pages/distroSettings/Context";
+import { omitTypename } from "utils/string";
 import { BaseTab } from "../BaseTab";
 import { getFormSchema } from "./getFormSchema";
-import { TabProps } from "./types";
+import { TabProps, ProviderFormState } from "./types";
 
 export const ProviderTab: React.FC<TabProps> = ({ distroData }) => {
   const initialFormState = distroData;
```

Comment:
[nit] No need to rename this var (even though I think we have been doing it unnecessarily in every tab 😂)